### PR TITLE
python3Packages.notus-scanner: init at unstable-2021-09-05

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Helper to create results from local security checks

https://github.com/greenbone/notus-scanner

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 137136` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>notus-scanner (python39Packages.notus-scanner)</li>
    <li>python38Packages.notus-scanner</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 137136` at 865e59d8 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>notus-scanner (python39Packages.notus-scanner)</li>
    <li>python38Packages.notus-scanner</li>
  </ul>
</details>

File: pkgs/development/python-modules/notus-scanner/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, paho-mqtt
+, poetry
+, psutil
+, pytestCheckHook
+, pythonOlder
```
Comment (samuela) [inline pkgs/development/python-modules/notus-scanner/default.nix:8]: nit: it would be nice to alphabetize

**Review by samuela — APPROVED** [review]

File: pkgs/development/python-modules/notus-scanner/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = pname;
+    rev = "049f9a5e6439e4e5113e3b8f30b25ead12d42a56";
+    sha256 = "1fjxyn8wg2kf6xy3pbh7d7yn20dk529p03xpqyz7s40n9nsxhnza";
+  };
+
+  nativeBuildInputs = [
+    poetry
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/notus-scanner/default.nix:26]: Why not poetry-core?

File: pkgs/development/python-modules/notus-scanner/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/notus-scanner/default.nix:8]: Hmm, there is `lib` and the rest is in alphabetical order.

File: pkgs/development/python-modules/notus-scanner/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/notus-scanner/default.nix]: Switched to `poetry-core`.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 137136` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>notus-scanner (python39Packages.notus-scanner)</li>
    <li>python38Packages.notus-scanner</li>
  </ul>
</details>

**Pull request merged**